code/modules/surgery/organs/stomach/external/_external_organs.dm

```diff
@@ -0,0 +1,219 @@
+/**
+* System for drawihg organs with overlays. Used as a replacement for the datumized species features, so that you can tear our a moths wings and stick it on a lizard
```

Comment:
```suggestion
* System for drawing organs with overlays. Used as a replacement for the datumized species features, so that you can tear our a moths wings and stick it on a lizard
```

Also, comments should explain what the code is *now*, not what it used to be.